### ISSUE TITLE
Lost emc values on items that previously had them

### ISSUE THREAD

**Issue by LazarusCaine** [sinkillerj/ProjectE#2204]
**Tickets that do not conform to this template will be closed without comment**

Exact ProjectE version (do not say "latest", "latest on Curse", or similar): ProjectE-1.18.2-PE1.0.1

Exact Forge version: forge-1.18.2-40.1.25

Link to crash log (please use a paste site, do not attach the .txt or paste the log inline): 

Steps to reproduce:

1. I updated to CraftTweaker-forge-1.18.2-9.1.143 and Mekanism-1.18.2-10.2.1.461

What I expected to happen:

I expected to add emc values for makanism items as well as keeping existing emc values on other items

What happened instead:

I got emc values for mekanism, but lost them for other items (ProjectE items, standard minecrtaft items such as forges and gold bars, etc.)

**Comment by LazarusCaine** [comment]
I found that not all ProjectE items have lost their emc values.  The Anti-Matter Relays still have their values.

**Comment by pupnewfster** [comment]
Based on https://github.com/mekanism/Mekanism/issues/7491 I presume this has to do with some mod that makes the emc values for various base items go to zero.

**Comment by LazarusCaine** [comment]
The only things I changed today were updating the 4 Mekanism Mods (Mekanism, Generators, Additions, Tools), updating CraftTweaker from 1.18.2-9.1.116 to 1.18.2-9.1.143 (I have since gone down to version 1.18.2-9.1.139 as this is the minimum I can run on the server now) and changing from forge-1.18.2-40.1.0 to forge-1.18.2-40.1.25.

**Comment by pupnewfster** [comment]
Yes, but given with just mekanism and projecte things still have EMC values, that just means that some other mod is likely adding some recipe that when ran through Mekanism's processing causes the EMC values to go to zero. As Mekanism 10.2.0 had a bug that made none of the custom recipe mappers it adds work. A better solution would be in `config/ProjectE/mappers.toml` (or something like that I forget the exact file name) to just disable the various mappers Mekanism adds if you don't want its machines to have input to output emc values calculated due to some other mod adding recipes that cause things to get down to a zero cost.

**Comment by LazarusCaine** [comment]
I updated Mekanism back to 1.18.2-10.2.1.461 and set all the Mekanism crafting mappers to false.  This did not correct the issue, the emc values I had before are still missing.

**Comment by pupnewfster** [comment]
Well then that is super odd as I can't remember any other changes on mek's end that would cause updating to make the emc values go away. You did reload after changing the config options if you were in game right?

**Comment by LazarusCaine** [comment]
Yes, I have reloaded after I made each of the changes while troubleshooting.
